subt/artf_node.py

```diff
@@ -104,11 +126,14 @@ def detect(self, image):
         assert self.width == img.shape[1], (self.width, img.shape[1])
 
         result = self.detector(img)
-        if len(result) > 0:
-            self.stdout(result)
-            report = result2report(result, self.depth)
-            if report is not None:
-                self.publish('artf', report)
-                self.publish('debug_artf', image)  # JPEG
+        result_cv = self.cv_detector(img)
+        if len(result) > 0 or len(result_cv) > 0:
```

Comment:
What are the types of `result` and `result_cv`? If they are simple python lists, `if result or result_cv` are more pythonic (I believe) expressions with the same result.

Reply:
Ok, why not